FileHandleWriter.go

```diff
@@ -109,9 +109,12 @@ func (this *FileHandleWriter) Flush() error {
 	op := this.Handle.File.FileSystem.RetryPolicy.StartOperation()
 	for {
 		err := this.FlushAttempt()
-		if IsSuccessOrBenignError(err) || !op.ShouldRetry("Flush()", err) {
+		if err != io.EOF || IsSuccessOrBenignError(err) || !op.ShouldRetry("Flush()", err) {
```

Comment:
could you put 'err != io.EOF' into ShouldRetry?

Reply:
I donot think file read should retry with `EOF` error.